[docs] Include usage instructions in README (#126)
repobee/repobee-sanitizer: README.md

going through files, looking for certain `REPOBEE-SANITIZER-<type>`
text-markers. The most simple usage consists of a `REPOBEE-SANITIZER-START` and
a `REPOBEE-SANITIZER-END` marker, where content between these two markers will
be removes, or "santitized" as it were.

## Install
Use RepoBee's plugin manager to install! 

First, make sure that you have the latest version of `Repobee` installed,
instructions can be found on the [Repobee website](https://repobee.org/).

To install `repobee-sanitizer`, simply run the command `repobee plugin install` in your terminal, and this will open up a bullet-list where you can select sanitizer. 

> > If `repobee-sanitizer` does not show up when running the `sanitize` command. Run `repobee plugin list`, this will present a list with all the availiable plugins. If you cannot find `repobee-sanitizer` in the list, try updating `repobee` using repobee `repobee manage upgrade`.

Author: The install section was stolen straight of [csvgrades](https://github.com/repobee/repobee-csvgrades) sooooo....

I'll change this later and ill have a look at the rest after my exams 👍

> > If `repobee-sanitizer` does not show up when running the `sanitize` command. Run `repobee plugin list`, this will present a list with all the availiable plugins. If you cannot find `repobee-sanitizer` in the list, try updating `repobee` using repobee `repobee manage upgrade`.

Reviewer: > The install section was stolen straight of [csvgrades](https://github.com/repobee/repobee-csvgrades) sooooo....

Pointing to [something you wrote yourself](https://github.com/repobee/repobee-csvgrades/pull/17) isn't the strongest argument for it being good :D. Reading that install section, I find it equally confusing.

> I'll change this later and ill have a look at the rest after my exams :+1:

Okay.

> > If `repobee-sanitizer` does not show up when running the `sanitize` command. Run `repobee plugin list`, this will present a list with all the availiable plugins. If you cannot find `repobee-sanitizer` in the list, try updating `repobee` using repobee `repobee manage upgrade`.

Author: For this one I just wrote something different from what i though in my head. Should have been "does not show up when running the `plugin list` command" or something but i think the way it is now is clearer.

## Commands
`repobee-sanitizer` supports two main commands:

> ### File

Reviewer: I'd recommend actually naming the command here

```suggestion
### The `sanitize file` command
```

> `repobee sanitize file <infile> <outfile>` performs the sanitization operation described below directly on a file `infile` and writes the output to `outfile` 

Reviewer: I'm missing an example invocation here. Something like:

```
$ repobee sanitize file input.txt sanitized.txt
```

The `--strip` flag can also be used to reverse the operation, instead removing all sanitizer syntax from the file.

> ### Repo

Reviewer: ```suggestion
### The `sanitize repo` command
```

> #### Repo root
> `repobee sanitize repo` performs the sanitization protocol on an entire repository. The directory you want to sanitize is either specified as the working directory `./` or it can be specified using the `--repo-root <path>` option.
> 

Reviewer: This should be the introduction to the section `sanitize repo` section, rather than a subsection (i.e. remove the `Repo roop` heading). It also gives a better overview if you start with an example command, and then add stuff to it. In short: explain by example!

````suggestion
`sanitize repo` performs the sanitization protocol on an entire repository, sanitizing the working directory of one branch and then committing the result to another branch. It's most basic usage looks like so.

```
$ repobee sanitize repo --target-branch main
```

Assuming that you're currently at the root of a Git repository on a branch other than `main`, this will sanitize the current branch and commit the result to `main`.
````

When that very basic usage has been explained, you can dive deeper into other options and concepts.

#### Branches
Another important feature is working with branches, `repobee-sanitizer` was essentially made to manage differing branches. For an example, a repo can have a `solutions` and a `main` branch, where the `solutions` branch contains an entire solution to an assignment, as well as `sanitizer` `markers` (specified below), when we then sanitize the `solutions` branch, this creates a slimmed-down version of our repo (that we would send to students), what we can then do is to specify which branch we want the sanitied version to end up on.

This is done using the `--target-branch <branch-name>` option. For example, if our repo is checked out to the branch `solutions` (that contains full solutions and `sanitizer` `markers`), Running:

> `repobee sanitize repo --target-branch main` 

Reviewer: To clarify that this is a shell command, I'd recommend writing it like so.

```
$ repobee sanitize repo --target-branch main
```
